### PR TITLE
[search] Added simple search http server.

### PR DESCRIPTION
**Pull request by ygorshenin**
@burivuh PTAL

### CODE REVIEW

**Comment by ygorshenin** [comment]
@burivuh PTAL

File: search/pysearch/run_search_server.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+        def fparam(name):
+            return float(sparam(name))
+
+        params = pysearch.Params()
+        try:
+            params.query = sparam('query')
+            params.locale = sparam('locale')
+            params.position = pysearch.Mercator(fparam('posx'), fparam('posy'))
+            params.viewport = pysearch.Viewport(pysearch.Mercator(fparam('minx'), fparam('miny')),
```
Comment (burivuh) [inline search/pysearch/run_search_server.py]: format please

File: search/pysearch/run_search_server.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+def main(args):
+    pysearch.init(args.r, args.m)
+    engine = pysearch.SearchEngine()
+    HTTPHandler.engine = pysearch.SearchEngine()
+
+    try:
+        print('Starting HTTP server on port', PORT)
+        server = HTTPServer(('', args.p), HTTPHandler)
+        server.serve_forever()
+    finally:
```
Comment (burivuh) [inline search/pysearch/run_search_server.py]: Actually, maybe this is unecessary

File: search/pysearch/run_search_server.py (hunk 1 above)
Comment (ygorshenin) [inline search/pysearch/run_search_server.py]: Done

File: search/pysearch/run_search_server.py (hunk 2 above)
Comment (ygorshenin) [inline search/pysearch/run_search_server.py]: Done

**Comment by ygorshenin** [comment]
PTAL

**Comment by ygorshenin** [comment]
Many thanks!

**Pull request merged**